SDCTA/src/components/Body.tsx

```diff
@@ -0,0 +1,56 @@
+import landscape from '../Images/unsplash_dsV4yvL5UJk.jpg';
```

Comment:
May be good to rename this file and its corresponding CSS file to something like LandingBody for less confusion (even if its put in its own separate folder)